### PR TITLE
feat(jest): 100% code coverage for Commands.test.ts

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory

**Which issue(s) this PR fixes** 🔨

AP-567

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: libraries/ui/Commands.ts [hunk 1]
```diff
@@ -11,6 +11,14 @@ export const commandPrefix = '/'
...
 export function containsCommand(text: string) {
   const cmd = text.split(' ')[0].replace(commandPrefix, '')
+  console.log(1, cmd, text)
+  console.log(
+    2,
+    text.charAt(0) === commandPrefix,
+    text.charAt(0),
+    commandPrefix,
+  )
+  console.log(3, cmd, text, cmd.match(/^[a-z0-9]+$/i), text.length === 1)
```
Comment (stavares843) [inline libraries/ui/Commands.ts:21]: do we need these console.logs? 🔨

File: libraries/ui/Commands.test.ts [hunk 2]
```diff
@@ -173,4 +173,41 @@ describe('Commands.isArgsValid', () => {
...
+  })
+
+  describe('Commands.hasCommandPreview', () => {
+    test('0', () => {
+      const object: string = '/a'
+      // Regex is [a-z0-9], hence everything that is in these is valid.
+      // Other condition that we need to consider so that it passes the test is that:
+      // it will return true if either the string has / at the start (text.charAt(0) === commandPrefix)
+      // AND
+      // the text is valid for the regex (/^[a-z0-9]+$/i) and has a length of 1 (/ab has a length of 2).
```
Comment (stavares843) [inline libraries/ui/Commands.test.ts:199]: can we add a multiline comment for these comments?

**Comment by stavares843** [comment]
minor comment

**Review by WanderingHogan — APPROVED** [review]
good after removing logs/multi line comment split

**Review by stavares843 — APPROVED** [review]

**Pull request merged**